data/korath/korath missions.txt

```diff
@@ -257,3 +257,604 @@ mission "Small Scale Delivery"
+			`	"Not understanding our society, they talked to the first fishers they found, who couldn't help them, because those fishers were slaves. To Korath, self-expression is a religion and almost a biological necessity. Slavery to them is as abhorrent as," she thinks for a moment, "consuming the skin of dead relatives is to us.`
+			`	"They made a deal: freedom for fish. The Korath provided rifles in secret, and the slaves drove their masters off the planet. Ex-Syndicate refugees who fled persecution to live here helped us form a functioning government. Now, we give juvenile fish to the Korath, and they assist us in industry, mining, food production, and defense of our systems.`
+			`	"We invited them - no, we asked them - to live on New Tortuga with us, to share our homes and be family. Sadly, their oppressors have forbidden them from setting foot on planets outside their exiled systems. Although we have occasional visits of technicians or doctors to New Tortuga, no Korath has stayed for more than a few days. Never has a military vessel touched down.`
+			`	"I fear this traumatic first contact has scarred their view of humans. They despise the Syndicate, for its anti-religion policies; and pirates, for enslavement. The vastness of our species makes them fearful, despite their technological superiority. I fear broader peaceful relations between humans and Korath will be many years away.`
+				goto check
+
+			label exile
+			action
+				set "Korath [F05]: The Rainbow Feather: exile"
+			`	"Before their exile, the Korath had a vast and thriving civilization that spanned many systems. Unfortunately, due to their very nature, they could not resolve differences easily. This led to a terrible war that left their civilization weakened.`
+			# Iris's explanation is extremely biased because her primary source
+			# is the Korath exiles. She is presenting a human interpretation of
+			# the Exiles' feelings about their own fate.
+			`	"The imperialist Drak race sent living weapons, the Archons, to conquer the Korath under the guise of protection. The Archons' antimatter weapons and seeming invincibility was too much, until the Korath gathered a larger fleet. When they managed to injure one Archon, the others revealed frightening powers. They tore open rifts in reality, simply swallowing up Korath ships. Hidden super-weapons in planetoids, intended as mines, were harmless, as the Archons shattered the planetoids and atomized the weapons. In every battle, all Korath military ships were destroyed, but the civilians were somehow unscathed.`
```

Comment:
> They tore open rifts in reality, simply swallowing up Korath ships.

The Pug did something similar to a ringworld, and that was way beyond the power of the Drak. I guess the scale is entirely different, though, and there is no guarantee these things are actually related. Or that the korath assessment of the Drak weapon was accurate.

Reply:
This is a human's interpretation of a Korath's story of their gestalt's racial memory of an Archon. It is intentionally vague and ominous.